humility-hiffy/src/lib.rs

```diff
@@ -184,8 +184,14 @@ impl<'a> HiffyContext<'a> {
     }
 
     fn definition(hubris: &'a HubrisArchive, name: &str) -> Result<HubrisGoff> {
+        // Depending on compiler version, definitions may be in either
+        // `self.definitions` or `self.variables`, so we'll check in both
+        // places.  Behavior depends on whether the compiler included DWARF /
+        // ELF debug info for the type of an otherwise-unused variable marked as
+        // `#[used]`; this changed in rust-lang/rust#140872
```

Comment:
you may also want to refer to https://github.com/oxidecomputer/hubris/pull/2169#issuecomment-3114482833 where i wrote a little thingy summarizing this.